screen_tabs.py

```diff
@@ -85,11 +86,26 @@ class ScreenTabsPrefs(AddonPreferences):
 		min = 0.5,
 		max = 1.0)
 
+	screen_text_length = IntProperty(
```

Comment:
If length is 0 then what user will see:
![default](https://user-images.githubusercontent.com/32243930/45771684-d1ed4f80-bc56-11e8-8664-9f277f17e443.png)

Reply:
If they use icons, they could see something like this: 
![z](https://user-images.githubusercontent.com/10817631/45772841-62537200-bc0e-11e8-85f5-accae32bff26.png)

Reply:
Yeah, but then we should alert user somehow otherwise it looks like a bug. It is very specific case scenarios but they need to be handled somehow. I don't have much time right now, but for example show special alert icon or display text until icon not set.